Automattic/wp-calypso
# MIgrate date-picker test from enzyme to testing-library

#### Changes proposed in this Pull Request

This PR refactors the following test files to use `@testing-library/react` instead of `enzyme`.

- `client/my-sites/backup/backup-date-picker/test/index.jsx`

Part of #63409.

#### Testing instructions

Verify tests still pass: `yarn run test-client client/my-sites/backup/backup-date-picker/test/index.jsx`
